editor/editor_node.cpp

```diff
@@ -1986,6 +1986,10 @@ void EditorNode::_edit_current() {
 				}
 			}
 
+			// Currently on script editor, so don't switch unless alt is pressed
+			else if (Object::cast_to<ScriptEditorPlugin>(editor_plugin_screen) && !Input::get_singleton()->is_key_pressed(KEY_ALT)) {
```

Comment:
An empty `if` statement looks weird to me. Surely there's another way to achieve the same effect?

Reply:
Hmm well I just want to break out of that particular if block, I dont want to `return` from the method.

I could potentially move it inside the next `else if()` as another nested if:
```cpp
else if (main_plugin != editor_plugin_screen ... blah blah) {
    if (/* test to see if it's script editor */ == false){
        // Run existing logic
    }
}
```

Or even just move it to the other `else if`... but I am hesitant to do so since it will become a massive `else if` condition that checks many conditions for many things - which is harder to understand and comment. Doing that will only make it confusing for the next person who comes along to see why this condition was added.